plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt

```diff
@@ -128,7 +140,11 @@ class CodeModernizerSession(
             return CodeModernizerStartJobResult.Disposed
         } catch (e: Exception) {
             val errorMessage = "Failed to start job"
-            LOG.error(e) { errorMessage }
+            if (e !is IOException) {
```

Comment:
Nice capture of an edge case with a comment

Reply:
We should also check here if `shouldStop.get()` is true before hiding the exception.